scrapegraphai/helpers/models_tokens.py

```diff
@@ -18,7 +18,9 @@
         "gpt-4-32k": 32768,
         "gpt-4-32k-0613": 32768,
     },
-
+    "azure": {
+        "gpt-3.5-turbo": 4096
+    },
```

Comment:
I think it would be cool to have also (gpt-4: 4096) and (gpt-4-turbo: 16k)

Reply:
sure.